Spokestack/TextToSpeech.swift

```diff
@@ -335,6 +349,20 @@ fileprivate struct TTSSSMLResponseData: Codable {
     let data: TTSSSMLResponseSynthesize
 }
 
+// Markdown
+fileprivate struct TTSMarkdownResponseURL: Codable {
```

Comment:
Couldn't you have one `TTSResponseURL` to rule them all since the field is named the same across the board (and should never change)?